src/integration-test/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectRevisionControllerIntegrationTest.java

```diff
@@ -204,8 +204,8 @@ public void testCreateWorkflowRevisionShouldReturnUnsupportedMediaTypeWithoutBod
     }
 
     @Test
-    public void testCreateWorkflowRevisionShouldReturnNotFoundIfNonExistingBucketId() {
-        given().pathParam("bucketId", 35434245)
+    public void testCreateWorkflowRevisionShouldReturnNotFoundIfNonExistingbucketName() {
+        given().pathParam("bucketName", 35434245)
```

Comment:
we keep 35434245 which is an integer or better to change it to a String?

Reply:
+1 to change the parameter value to a proper name